machine.md

```diff
@@ -252,6 +252,33 @@ spec:
       persistentVolumeClaim:
         claimname: myclaim
 ```
+## Resources Requests and Limits
+
+An optional Resources requests can be specified by the users to
+allow the scheduler to make a better decision in finding the most suitable Node
+to place the Virtual Machine on.
+
+```
+    resources:
+      requests:
+        memory: "1Gi"
+        cpu: "2"
+      limits:
+        memory: "2Gi"
+        cpu: "1"
+```
+
+#### CPU
+Specifying CPU limits will determine the amount of time Virtual Machine CPUs
+can execute on the assigned resources when there is a competition for CPU
+resources.
+
+#### Memory Overhead
+Various virtual machine resources, such as a video adapter, IOThreads, and
+supplementary system software, consume additional memory from the Node, beyond
+the requested memory intended for the guest OS consumption. In order to provide
+a better estimate for the scheduler, this memory overhead will be calculated
+and added to the requested memory.
```

Comment:
The sentence blow is in the wrong section now, it should stay with the features section.